main/main.cpp

```diff
@@ -641,6 +641,9 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	bool found_project = false;
 #endif
 
+	String defaultRenderer = "";
+	String rendererHints = "";
```

Comment:
We use snake case for identifiers.

Reply:
The only reason I used this gross (in my opinion) code style is because my local declarations didn't compile on most platforms since a goto jumps over this code and that's illegal in gcc.  It isn't my fault that there is a goto to begin with, for the record.  

The extra setting it to "" is intentional to represent the conscious choice of using these values in this case, regardless of what the default is.   I was hoping that together with the comment this would help the next developer decide what should happen in this case (a case that never happens today since building with no driver makes no sense.)

Regarding the snake case, are you blocking this PR for that?

Reply:
> The only reason I used this gross (in my opinion) code style is because my local declarations didn't compile on most platforms since a goto jumps over this code and that's illegal in gcc. It isn't my fault that there is a goto to begin with, for the record.

Ah I see, that doesn't compile indeed.

> The extra setting it to "" is intentional to represent the conscious choice of using these values in this case, regardless of what the default is. I was hoping that together with the comment this would help the next developer decide what should happen in this case (a case that never happens today since building with no driver makes no sense.)

There are 6839 String declarations in the codebase. Them defaulting to an empty String *is* already a conscious decision and it's redundant to specify it again.

> Regarding the snake case, are you blocking this PR for that?

Yes, we have a code style and it's important to respect it. We all work together on this codebase and the code needs to be familiar to all contributors.

Reply:
> There are 6839 String declarations in the codebase. Them defaulting to an empty String _is_ already a conscious decision and it's redundant to specify it again.

That's an opinion and I disagree.  Since it is your code base of course you can do as you wish.

I was trying to communicate with the next developer down the line and since the static analyzer in the compiler removes that second initialization anyway, I didn't see any downside.   I will of course change it to suit your preference, but I won't pretend like I don't know what I am doing.

Reply:
snakes have been added